include/cuspv/cuda_runtime_api.h

```diff
@@ -0,0 +1,4 @@
+// in CUDA, this header has the C definitions;
+// cuda_runtime.h has all of this + C++ definitions
+
+#include <cuda_runtime.h>
```

Comment:
Yes, for now all three headers are essentially the same. It might make sense to separate C from C++ declarations (like templates) later, but only if required by some user / project.